### ISSUE TITLE
Lost Connection 

### ISSUE THREAD

**Issue by scooter306** [jeatheak/Mitsubishi-WF-RAC-Integration#90]
My MHI devices lost connection at midnight as I have seen in HA-Logs. 
at the same time my Smart M-Air-App gives me connection error. Is it only me or is the MHI-Server down?

**Comment by naimadfpv** [comment]
same here

**Comment by scooter306** [comment]
The App Got ein Update vier Days a go. Theme like there. Where some Breaking Changes the Way of Connection to the API.

**Comment by naimadfpv** [comment]
I installed new core update today and the integration is working again. 

Can’t really tell if it’s related to the new core update because I only use this integration on my workplace where I’m not everyday.

**Comment by scooter306** [comment]
Installed all updates. Had connection a few minutes, but again HA can’t connect to the ACs

**Comment by sean-2024** [comment]
I've observed an issue where if I restart HA quite often it doesn't manage to connect to all 4 units I have added in. Sometimes it'll connect to just 1 or 2. It appears once this happens, the ones it didn't manage to connect to during start up are never added in automatically.

I end up disabling 3 of them, restarting HA then slowly re-enabling them. 

Not sure if this is similar to the comments above or not. It feels like there may be room for improvement in that some retrying would be nice to establish the connection if it initially fails.
